bundles/io/org.eclipse.smarthome.io.transport.dbus/build.properties

```diff
@@ -0,0 +1,9 @@
+bin.includes = META-INF/,\
+               OSGI-INF/,\
+               .,\
+               about.html,\
+               lib/libdbus-java-2.7.jar,\
+               lib/unix-0.5.jar

```

Comment:
> The lib I think comes from the same author as dbus Java.

And is thus under the same licenses, so that we cannot add it to ESH. You could possibly take it out as a fragment, which we then put in OH2.

> This one is the Unix socket library which uses JNI

They should then certainly also be part of the fragment. You will imho actually need different files for the different architectures - x86, x86_64, ARM, ARM64, etc.